### DISCUSSION TITLE
Mysterious pink object for first person controller

### DISCUSSION THREAD

**Discussion by ammlt** [pokepetter/ursina#599] [Q&A]
Hi guys! This is my first time using ursina and I was trying to make a simple first person game following tutorials.

However once I add the first person controller in the code there is a pink object in the middle of the screen and I can't find where did the code added in the object. 
I have tried disable everything except the first person controller but the pink object it is still there.
Also, except i could walk around one ground just fine, but I cannot turn the camera around with my cursor.

I wonder if there is any problem in my ursina sourcecodes or any setting problems causing it? I tried uninstall and reinstalling ursina but the problem still persists and now I don't know what else can I do to solve the issue.

**Comment by pokepetter** [comment]
FirstPersonController has a variable called `cursor`, that's what you're looking for. You could disable it or change how it look, however you wish.
